edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Http/HttpProtocolHead.cs

```diff
@@ -3,37 +3,37 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Http
 {
     using System.Threading;
     using System.Threading.Tasks;
-    using Microsoft.AspNetCore.Hosting;
     using Microsoft.Azure.Devices.Edge.Hub.Core;
     using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Hosting;
```

Comment:
Why this change?

Reply:
Check out Hosting.cs changes, as it is recommended to replace WebHostBuilder by HostBuilder (https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#hostbuilder-replaces-webhostbuilder).